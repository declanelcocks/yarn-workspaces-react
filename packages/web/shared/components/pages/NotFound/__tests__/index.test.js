import React from 'react'
import { shallow } from 'enzyme'

import NotFound from '../index'

describe('<NotFound />', () => {
  test('renders', () => {
    const staticContext = {}
    const wrapper = shallow(<NotFound staticContext={staticContext} />)
    expect(wrapper).toMatchSnapshot()
    expect(staticContext.missed).toBeTruthy()
  })
})
