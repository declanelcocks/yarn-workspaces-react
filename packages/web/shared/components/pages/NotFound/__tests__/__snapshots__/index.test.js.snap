// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders 1`] = `
<div>
  Sorry, that page was not found.
</div>
`;
